File: mboh.py
#deklarasikan huruf yang akan diganti
a = "aueo" 
#minta user buat inputin teks
b = input("Masukin teks : ") 
#loop sebanyak jumlah huruf yang akan diganti
for c in a:
  #anti-si-sapi input karakter biar nggak berantakan, sekalian aja dibikin huruf kecil semua
  #gantikan semua karakter yang dideklarasikan sebelumnya, ke 'i'
  b = b.lower().replace(c, "i")
  #tampilin hasilnya
  print ("Hasilnya : " + b)

File: mboh.php
<?php
//deklarasikan huruf yang akan diganti
$a = str_split('aueo');
//minta user buat inputin teks (karena PHP itu server side
//kita coba pakai $_GET supaya input bisa langsung dari parameter alamat
$b = strtolower($_GET['teks']);
//loop sebanyak jumlah huruf yang akan diganti
foreach ($a as $c)
//gantikan semua karakter yang dideklarasikan sebelumnya, ke 'i'
{ $b = str_replace($c,'i',$b); };
//tampilin hasilnya
echo "Hasilnya : ".$b;
?>